packages/osd-stylelint-plugin-stylelint/src/utils/get_rules_from_config.ts

```diff
@@ -27,3 +27,13 @@ export const getSelectorRule = (rules: any, rule: any) => {
 
   return undefined;
 };
+
+export const getPropertyRule = (arr: Record<string, { approved?: string[] }>, decl: any) => {
```

Comment:
nit: `arr` is a little bit of misname as it it's a record

Reply:
Thanks, this was from an old implementation. Updated